newIDE/app/src/MainFrame/index.js

```diff
@@ -2330,14 +2444,18 @@ const MainFrame = (props: Props) => {
         })}
       </LeaderboardProvider>
       <CommandPalette ref={commandPaletteRef} />
-      <LoaderModal show={showLoader} />
+      <LoaderModal
+        show={showLoader}
+        progress={fileMetadataOpeningProgress}
+        message={fileMetadataOpeningMessage}
+      />
       <HelpFinder
         open={helpFinderDialogOpen}
         onClose={() => openHelpFinderDialog(false)}
       />
       <Snackbar
         open={state.snackMessageOpen}
-        autoHideDuration={3000}
+        autoHideDuration={5000}
```

Comment:
Maybe it's because I'm on dev, lambda's are on cold start, but yeah it feels long. I could also add the loader modale that displays messages.

Reply:
I think 3s should be enough on prod yeah. I'm not sure what you mean by loader modale?

Reply:
<img width="304" alt="Capture d’écran 2022-07-28 à 11 07 48" src="https://user-images.githubusercontent.com/32449369/181467754-eb52d3b3-8e0f-4dbf-9901-2db2a198ec31.png">

I'm taking about this

Reply:
I didn't see your conversation above about the time it takes to duplicate.
Ideally we'd close this snackbar manually instead of having a timeout, but probably fine to go with 5 seconds for the moment yes 👍

Reply:
I changed it back to 3s